### ISSUE TITLE
General snippet textarea label and input don't match

### ISSUE THREAD

**Issue by maya** [18F/tock#532]
On the general snippet textarea, the label "for" and input "id" don't match. They should be the same so they're explicitly associated with each other per W3C F68 4.1.2 https://www.w3.org/TR/WCAG20-TECHS/F68.

```
<div class="entry-notes" id="entry-notes">
  <label for="notes">Notes:</label>
  Please provide a snippet about how you spent your time.
  <textarea class="entry-notes-text" cols="40" id="id_timecardobject_set-1-notes" name="timecardobject_set-1-notes" rows="10"></textarea>
</div>
```

**Comment by maya** [comment]
Closing in favor of consolidated issue: #533.
